governance/02-issues-workflow.md

```diff
@@ -25,13 +25,15 @@ The default labels provided by GitHub are as follows:
 
 Here are the custom labels introduced by the Kyma team. The labels colors are provided in brackets in Hex code: 
 
-* `WIP` (#ECF44F) indicates that an issue is already in progress.
+* `WIP` (#ECF44F) indicates that an issue is not ready for further processing.
 * `decision` (#ED635E) indicates that an issue is related to a decision.
 * `priority/critical` (#FB0104) indicates the top-priority of a given issue.
 * `area/{CAPABILITY_NAME}` (#3CB913) indicates which capabilities are related to a given issue. You can assign more than one `area` label an issue.
 * `security/{SEVERITY}` (#2D51F9) indicates a security issue based on its [CVSSv3](https://www.first.org/cvss/calculator/3.0) severity, either `low`, `medium`, `high`, or `critical`.
 * `sig/{SIG_NAME}` (#E99694) indicates which [Special interest group (SIG)](/collaboration/#overview-overview) identified the issue and is responsible for further follow-up on the issue.
 * `wg/{WG_NAME}` (#E99694) indicates which [Working group (WG)](/collaboration/#overview-overview) identified the issue and is responsible for further follow-up on the issue.
+* `area/quality` (#e99695) indicates that an issue is related to a quality topics.
```

Comment:
Is such a label already created? For now, quality labels are like this: quality/{QUALITY_NAME}. If a label `area/quality` is going to be created, maybe we can move the description to line 31 so that both `area` labels are next to each other? :)

Reply:
area/quality label is already created, I will move it up next to teh other area label